packages/http-client-csharp/emitter/src/emitter.ts

```diff
@@ -114,8 +114,7 @@ export async function $onEmit(context: EmitContext<CSharpEmitterOptions>) {
       );
       logger.info(`Checking if ${csProjFile} exists`);
 
-      const emitterPath = options["emitter-extension-path"] ?? import.meta.url;
```

Comment:
After we remove this, it means the project root is always the MTG, not the sub-emitter. Then we can't get the sub-generator during generator loading.